Tools/LambdaTestTool/README.md

```diff
@@ -225,16 +227,26 @@ The path to the .NET Core 3.1 entry assembly is:
 
```

Comment:
Basically we should do a search and replace in the README file for `WebTester` and replace with `BlazorTester`

Reply:
Thanks. I have made the necessary changes. Also rephrased the Note(s) to mention about WebTester for `.NET Core 2.1` and `.NET Core 3.1` (version earlier than `0.11.0`). This would highlight `Amazon.Lambda.TestTool.BlazorTester.dll`.